units/src/amount/result.rs

```diff
@@ -172,6 +182,11 @@ crate::internal_macros::impl_op_for_references! {
 
         fn div(self, rhs: u64) -> Self::Output { self.and_then(|lhs| lhs / rhs) }
     }
+    impl ops::Div<Amount> for Amount {
+        type Output = u64;
+
+        fn div(self, rhs: Amount) -> Self::Output { self.to_sat() / rhs.to_sat() }
```

Comment:
I think this should round up instead of int division _or_ a round_up should be added.

Reply:
I would expect `ops::Div` to be truncating (can be seen in `fee` also). You could raise an issue if you think we should add an additional method on the amount types though.

Reply:
Agreed. The `/` operator should definitely round down.

Reply:
> Agreed. The / operator should definitely round down.

Alright, although I've seen cases in practice where what's needed is to round-up.  An option to roundup would be nice but that doesn't need to happen here.